openedx/core/djangoapps/catalog/utils.py

```diff
@@ -135,3 +138,41 @@ def get_programs_with_type(types=None):
             programs_with_type.append(program_with_type)
 
     return programs_with_type
+
+
+def get_catalog_course_runs():
```

Comment:
Just name this function `get_course_runs`. It's in module with other catalog utilities; putting catalog in the name here is unnecessarily redundant.